### DISCUSSION TITLE
The Information Theory, Patter Recognition, and Neural Networks - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#339] [Announcements]
# The Information Theory, Patter Recognition, and Neural Networks - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/The_Information_Theory_Pattern_Recognition_and_Neural_Networks/

**Comment by baichuanzhou** [comment]
捉个虫哦，应该是Pattern Recognition

**Reply by taylover2016** [reply]
hhh确实，当时好像少打了。打算下次提交pr的时候一并改一下。

**Comment by pengzhangzhi** [comment]
这门课最近听了一下，感觉课一般。书写的不错。如果你对贝叶斯推断感兴趣可以去读一下那本书。

**Reply by taylover2016** [reply]
书确实是经典之一，不过当时就是想着看下网课，水过学校考试 233 课其实已经够了
